client/src/main/java/io/pravega/client/control/impl/Controller.java

```diff
@@ -364,6 +365,14 @@
      */
     CompletableFuture<Transaction.Status> checkTransactionStatus(final Stream stream, final UUID txId);
 
+    /**
+     * Get list of TransactionInfo for the Stream.
```

Comment:
More details needed here. Which ones, how many, etc.

Reply:
Updated the method details.